tools/crm_attribute.c

```diff
@@ -732,10 +754,9 @@ main(int argc, char **argv)
         attrd_opts = pcmk__node_attr_remote;
     }
 
-    if (((options.command == 'v') || (options.command == 'D') || (options.command == 'u'))
-        && try_attrd
-        && (send_attrd_update(options.command, options.dest_uname, options.attr_name,
-                              options.attr_value, options.set_name, NULL, attrd_opts) == pcmk_rc_ok)) {
+    if (try_ipc_update(try_attrd) &&
```

Comment:
I feel like we don't need try_attrd at all anymore, it can all be wrapped into try_ipc_update()